src/calculate-fps.spec.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import calculateFPS, { getTotalHiddenTime } from './calculate-fps'
+import {equal} from 'assert'
+
+describe('calculate-fps', () => {
+  describe('calculateFPS', () => {
+    it('should calculate the fps', () => {
+      const frames = [200, 400, 600, 800, 1000]
+
+      equal(calculateFPS(frames, []), 5)
+    })
+
+    it('should calculate the fps taking into account the hidden delta', () => {
+      const frames = [200, 400, 600, 800, 1000]
+      const visibilityEvents = [[false, 0], [true, 300]]
+
+      equal(calculateFPS(frames, visibilityEvents), 8)
+    })
+  })
+
+  describe('getTotalHiddenTime', () => {
+    it('should calculate the amount of time that it spent hidden', () => {
+      const visibilityEvents = [
+        [false, 1000],
```

Comment:
Just for the sanity of the reader of these tests, could you do this at the top and replace all the `true`/`false` in the tests. When you're not completely familiar with the API, it's hard to understand what the visibility event array actually means.

```
const BECAME_VISIBLE = true
const BECAME_HIDDEN = false
```